docs/getting-started/roadmap.md

```diff
@@ -1,20 +1,32 @@
 # Roadmap
 
-## Release 1.0 onwards
+This document defines a high level roadmap for KubeEdge development.
 
-KubeEdge will provide the fundamental infrastructure and basic functionality for IOT/Edge workloads. This includes: 
+The [milestones defined in GitHub](https://github.com/kubeedge/kubeedge/milestones) defined in GitHub represent the most up-to-date plans.
 
-- Istio-based service mesh across Edge and Cloud where micro-services can communicate freely in the mesh.
+KubeEdge will provide the fundamental infrastructure and basic functionality for IOT/Edge workloads.
+
+KubeEdge 1.0 is our current stable branch. The roadmap below outlines new features that will be added to KubeEdge.
+
+## Release 1.1
+- Replace data exchange format between cloud and edge from json to protobuf.
+- Support reliable message delivery from cloud to edge.
+- Evaluate gRPC for cloud to edge communication.
+- Support PV/PVC for stateful workloads at the edge.
+- Support ingress at edge.
 - Enhance performance and reliability of KubeEdge infrastructure.
+- Upgrade Kubernetes dependencies in vendor to v1.15.
+- Migrate to Go module for dependency management.
+- Improve contributor experience by defining project governance policies, release process, membership rules etc.
+- Improve the performance and e2e tests with more metrics and scenarios.
+
+## Future
+- Support edge-cloud communication using edgemesh.
+- Add Layer 4 proxy support in edgemesh.
+- Istio-based service mesh across Edge and Cloud where micro-services can communicate freely in the mesh.
 - Enable function as a service at the Edge.
-- Support more types of device protocols to Edge nodes such as AMQP, BlueTooth, ZigBee, etc.
+- Support more types of device protocols such as OPC-UA, Zigbee.
 - Evaluate and enable much larger scale Edge clusters with thousands of Edge nodes and millions of devices.
 - Enable intelligent scheduling of applications to large scale Edge clusters.
 - Data management/analytics framework.
-
-## Release 0.3
```

Comment:
Do we need to keep the previous things here?

Reply:
I feel roadmap should list future work, so past release info need not be maintained. This info can be retrieved from the Github releases.